src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1307,7 +1329,11 @@ private void walk(TypedVar symbol, String emitName) {
           visitTypeValueAlias(symbol.getName(), (EnumElementType) registryType);
           return;
         }
-        visitVarDeclaration(getUnqualifiedName(emitName), type);
+        if (aliasMap.containsKey(emitName)) {
```

Comment:
This is my overlooking, but it would be nice to have a comment describing when do we hit this branch - 'missing provide'?

Reply:
I don't quite follow, but added a comment.